relay-server/src/services/store.rs

```diff
@@ -1361,6 +1361,8 @@ struct SpanKafkaMessage<'a> {
     #[serde(default)]
     organization_id: u64,
     #[serde(default, skip_serializing_if = "Option::is_none")]
+    origin: Option<&'a str>,
```

Comment:
Is the `origin` guaranteed to be ASCII? If not, we should probably use `#[serde(borrow)] Cow<'a, str>` to not fail deserialization.

https://serde.rs/lifetimes.html#borrowing-data-in-a-derived-impl

Reply:
Good catch!

Reply:
We could go either way, although, it'd rather at least enforce it's a string.